frontend/nuxt.config.js

```diff
@@ -71,6 +71,11 @@ export default {
         domain: process.env.PRODUCTION_DOMAIN,
     },
 
+    // Disable Static Asset Path Prefix: https://nuxtjs.org/docs/directory-structure/static/#static-asset-prefixs
+    static: {
+        prefix: false,
+    },
```

Comment:
Why do you need to disable it? I don't think we can do that, since this folder contains favicon and Terms and Conditions pdf

Reply:
You're right, I thought the access to the folder was the problem, but I think it was only the path to the file, since those 2 things work in staging & prod.

The problem might've been that I used `src: 'js/HeapIO.js'` instead of `src: '/js/HeapIO.js'`